src/Command/Ssh/SshKeyCreateCommand.php

```diff
@@ -152,8 +155,7 @@ protected function determinePassword(InputInterface $input, OutputInterface $out
    */
   protected function validatePassword($password) {
     $violations = Validation::createValidator()->validate($password, [
-      new Length(['min' => 5]),
-      new NotBlank(),
+      new Length(['min' => 0])
```

Comment:
I would prefer to leave this validation here and conditionally bypass it when inside of an IDE

Reply:
Or better yet, when we know that we are calling it from the wizard command.